src/main/java/com/tutorhub/s3/client/service/AwsS3Service.java

```diff
@@ -19,30 +17,60 @@ public class AwsS3Service {
   private final String bucket = "tutorhub.storage";
   private final AmazonS3 s3;
 
-  private final Map<ContentType, Function<InputStream, byte[]>> methodByFileType = new HashMap<>();
-
-  @PostConstruct
-  protected void init() {
-    methodByFileType.put(ContentType.IMAGE, this::extractImage);
-    methodByFileType.put(ContentType.VIDEO, this::extractVideo);
-  }
-
   @SneakyThrows
-  public Object findFile(String fileName, ContentType contentType) {
+  public Object find(String fileName) {
     S3Object object = s3.getObject(bucket, fileName);
 
-    var function = methodByFileType.get(contentType);
+    InputStream content = object.getObjectContent();
 
-    return function.apply(object.getObjectContent());
+    return IOUtils.toByteArray(content);
   }
 
-  @SneakyThrows
-  private byte[] extractImage(InputStream inputStream) {
-    return IOUtils.toByteArray(inputStream);
+  public boolean checkIfExists(String filename) {
```

Comment:
I suggest to not use word `check` in boolean method name, because we can misunderstood the purpose of method.

Prefer for booleans names: `exists**()`, or `is***()`, here we know that method only checks existence, but in `checkIfExists` we can think about some exception throwing

Reply:
Fixed